src/chrome/content/rules/5ch.xml

```diff
@@ -0,0 +1,45 @@
+<ruleset name="5ch.net">
+        <target host="5ch.net" />
+        <target host="*.5ch.net" />
```

Comment:
Please do not use wildcard targets if possible.

Reply:
Is it better this way?
`<target host="\w+.5ch.net" />`

**5ch** (formerly 2ch) is the largest forum in Japan. 5ch has fewer servers. But it's fluid rather than fixed.

The former 2ch (not **5ch**) has grown to over 100 subdomains: [sample](http://qb5.5ch.net/)

Reply:
The `host` attribute is not a regex, so this won't work. If it's actually impossible to list all subdomains, a wildcard is acceptable.

Reply:
I think a wildcard is good.

erhaps they are worried about connection errors with HTTPS. But, SSL has been used since the previous 2ch (current 5ch). The web operator is SSL-enabled, can also connect to both via HTTP. It won't cause any problems.

On the other hand, the official smartphone loading program (still output HTTP) has been stalled in development. Programmers are not active. And external third party web tools also.